src/scss/hpe/_hpe.defaults.scss

```diff
@@ -46,7 +47,7 @@ $brand-graph-colors: (#FF8D6D, #877B75, #2AD2C9, #614767, #617D78);
 
 html,
 .brand-font {
```

Comment:
All you need to do is add ', .brand-font' after the 'html' selector in _base.font.scss. The theme dynamism will come from the setting of $brand-font-family.